src/collision.h

```diff
@@ -36,37 +36,20 @@ enum CollisionType
 
 struct CollisionInfo
 {
-	enum CollisionType type;
-	v3s16 node_p; // COLLISION_NODE
-	bool bouncy;
+	CollisionInfo() {}
+	CollisionType type = COLLISION_NODE;
+	v3s16 node_p = v3s16(-32768,-32768,-32768); // COLLISION_NODE
```

Comment:
the value fits in the container, i don't really know it's the original behaviour, i think we can keep this as is as it's not a PR which should trigger regression by changing values :)